dependabot.yml

```diff
@@ -0,0 +1,14 @@
+# Basic dependabot.yml configuration file
+
+version: 2
+updates:
+  # Enable version updates for maven
+  - package-ecosystem: "maven"
+    directory: "/"
+    target-branch: "dev"
+    reviewers:
+      - "i-Alex"
+      - "abi87"
```

Comment:
abi87 is Alberto Benegiamo. Do we need him or Alberto Garoffolo (albertog78)?

Reply:
Added Albene because don't like to have only one person as reviewer (and Albene volunteered).
If you think is necessary i can add also Algaro